components/scrapd-home-view/component.js

```diff
@@ -72,9 +76,10 @@ const ScrapdHomeView = () => {
     <div>
       <ScrapdHero
         title="scrAPD"
-        subtitle1="Safety data, collected automatically for you"
-        subtitle2="ScrAPD scrapes the web for traffic fatality data, making it accessible to all"
-        url="https://res.cloudinary.com/ab91/image/upload/v1567305230/scrAPD%20Viz/hero_image.png"
+        subtitle1="Austin’s latest traffic fatality data, in one spot"
```

Comment:
Maybe we can have a tagline a tad less positive. We are talking about deaths.

Reply:
What do you think of: "Visualize Austin traffic fatalities from another angle"?

This is the tagline we use for the repo.

Reply:
Complete

Reply:
Changed to "Visualize Austin traffic fatalities from another angle"